_std/macros/atom_properties.dm

```diff
@@ -261,6 +261,10 @@ To remove:
 #define PROP_MOB_EQUIPMENT_MOVESPEED(x) x("equipment_movespeed", APPLY_ATOM_PROPERTY_SUM, REMOVE_ATOM_PROPERTY_SUM)
 #define PROP_MOB_EQUIPMENT_MOVESPEED_SPACE(x) x("equipment_movespeed_space", APPLY_ATOM_PROPERTY_SUM, REMOVE_ATOM_PROPERTY_SUM)
 #define PROP_MOB_EQUIPMENT_MOVESPEED_FLUID(x) x("equipment_movespeed_fluid", APPLY_ATOM_PROPERTY_SUM, REMOVE_ATOM_PROPERTY_SUM)
+#define PROP_MOB_TALISMAN_SWIFTNESS(x) x("art_talisman_movement_mod", APPLY_ATOM_PROPERTY_SUM, REMOVE_ATOM_PROPERTY_SUM)
+#define PROP_MOB_TALISMAN_BRUTE_REDUCTION(x) x("art_talisman_brute_reduction", APPLY_ATOM_PROPERTY_SUM, REMOVE_ATOM_PROPERTY_SUM)
+#define PROP_MOB_TALISMAN_BURN_REDUCTION(x) x("art_talisman_burn_reduction", APPLY_ATOM_PROPERTY_SUM, REMOVE_ATOM_PROPERTY_SUM)
+#define PROP_MOB_TALISMAN_TOX_REDUCTION(x) x("art_talisman_tox_reduction", APPLY_ATOM_PROPERTY_SUM, REMOVE_ATOM_PROPERTY_SUM)
```

Comment:
Do these need to be talisman specific?

Reply:
Could you clarify? It's property sum so should be fine with multiple talismans?

Reply:
Could these be applicable outside of talismans? Is so in not sure the word talisman needs to be in the name.

Reply:
Could these be applicable outside of talismans? Is so in not sure the word talisman needs to be in the name.

Reply:
Could be yeah. I can change if you'd like, but name can be changed whenever so dunno